spec/helpers/action_view_extension_spec.rb

```diff
@@ -105,6 +105,11 @@
       let(:monetizable_object){ false }
       it { is_expected.to eq "00" }
     end
+
+    context 'with a negative monetizable object' do
+      let(:monetizable_object){ Money.new(-125) }
```

Comment:
Minor presentation fix to clarify the cents part here:

```suggestion
      let(:monetizable_object){ Money.new(-1_25) }
```